src/app/models/feedback.py

```diff
@@ -0,0 +1,63 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.sql.expression import select
+from sqlalchemy.orm import aliased
+from sqlalchemy.engine import Result
+from src.core.database.models.feedback import DataModelPredictionFeedback as dbFeedback
+from src.core.database.models.player import Player as dbPlayer
+from src.app.views.response.feedback import PredictionFeedbackResponse
+import logging
+
+
+class AppModelFeedback:
```

Comment:
rename to Feedback

Reply:
will do, was hardcore debugging, and feedback being all the class names was not helping

Reply:
resolved by [b9f3d53](https://github.com/Bot-detector/public-api/pull/25/commits/b9f3d5308b1e9f4c682ef510fceefec78b626b49)